tests/Integration/Integration.Tests/DailyEuReportsModuleTests/ContextBase.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace Linn.Stores2.Integration.Tests.DailyEuReportsModuleTests
+{
+    using Linn.Common.Reporting.Resources.ResourceBuilders;
+    using Linn.Stores2.Domain.LinnApps.Reports;
+    using Linn.Stores2.Facade.Services;
+    using Linn.Stores2.Integration.Tests.Extensions;
+    using Linn.Stores2.IoC;
+    using Linn.Stores2.Service.Modules;
+    using Microsoft.Extensions.DependencyInjection;
+    using NUnit.Framework;
+    using System.Net.Http;
+
+    using NSubstitute;
+
+    public class ContextBase
+    {
+        protected HttpClient Client { get; set; }
+
+        protected HttpResponseMessage Response { get; set; }
+
+        protected TestServiceDbContext DbContext { get; private set; }
```

Comment:
do you need all of this TestServiceDbContext stuff for these tests? Looks like the domain is just mocked out anyway so no 'real' persistence code will ever be in play